stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -33,7 +33,16 @@ spec:
         image: "{{ .Values.image.fluent_bit.repository }}:{{ .Values.image.fluent_bit.tag }}"
         imagePullPolicy: "{{ .Values.image.pullPolicy }}"
         env:
+{{- if .Values.backend.es.http_passwd_secret }}
+          - name: HTTP_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.backend.es.http_passwd_secret }}
+                key: {{ .Values.backend.es.http_passwd_secret_key }}
+{{- end }}
+{{- if or .Values.env (not .Values.backend.es.http_passwd_secret) }}
```

Comment:
I'm not following the logic here...

If values is nothing, and http_password_secret is nothing, still toYaml .Values.env?

Shouldn't this be this:
{{- if .Values.env }}

?